controller/scheduler/swarm.py

```diff
@@ -1,12 +1,10 @@
 import re
 import time
-
```

Comment:
These blank lines shouldn't be removed. From [PEP8](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:
- standard library imports
- related third party imports
- local application/library specific imports

> **You should put a blank line between each group of imports.**